packages/stylelint-config-scss/package.json

```diff
@@ -20,20 +20,20 @@
     "test": "jest"
   },
   "peerDependencies": {
-    "stylelint": ">=14.0.0"
+    "stylelint": ">=15.0.0"
   },
   "dependencies": {
     "stylelint-color-format": "1.1.0",
-    "stylelint-config-recommended-scss": "11.0.0",
-    "stylelint-config-standard": "29.0.0",
-    "stylelint-csstree-validator": "2.1.0",
+    "stylelint-config-recommended-scss": "12.0.0",
+    "stylelint-config-standard": "33.0.0",
+    "stylelint-csstree-validator": "github:bryanjtc/stylelint-validator#upgrade-package",
     "stylelint-declaration-block-no-ignored-properties": "2.7.0",
     "stylelint-declaration-strict-value": "1.9.2",
     "stylelint-high-performance-animation": "1.8.0",
     "stylelint-no-indistinguishable-colors": "github:bryanjtc/stylelint-no-indistinguishable-colors#upgrade-package",
     "stylelint-order": "6.0.3",
     "stylelint-scss": "5.0.0",
-    "stylelint-selector-tag-no-without-class": "2.0.5",
+    "stylelint-selector-tag-no-without-class": "github:bryanjtc/stylelint-selector-tag-no-without-class#upgrade-package",
```

Comment:
You had to fork it? If so, we can just remove this rule - it is better in the long run.